host/tests/dsClasses/dsHdmiIn.py

```diff
@@ -0,0 +1,445 @@
+
+    def initialise(self, device_type:int=0, connectionCBFile:str="", signalchangeCBFile:str="", statusCBFile:str="",
+                   videomodeCBFile:str="", allmCBFile:str="", avlatencyCBFile:str="", avicontentCBFile:str=""):
+        """
+        Initializes the device settings HdmiIn module.
+
+        Args:
+            device_type (int, optional): 0 - sink device, 1 - source device. Defaults to sink.
+
+        Returns:
+            None
+        """
+        promptWithAnswers = [
+                {
+                    "query_type": "direct",
+                    "query": "Select Device Type[0: Sink, 1: Source]:",
+                    "input": str(device_type)
+                },
+                {
+                    "query_type": "direct",
+                    "query": "Enter file name with path to log connection status callbacks:",
+                    "input": connectionCBFile
+                },
+                {
+                    "query_type": "direct",
+                    "query": "Enter file name with path to log  signal change callbacks:",
+                    "input": signalchangeCBFile
+                },
+                {
+                    "query_type": "direct",
+                    "query": "Enter file name with path to log status change callbacks:",
+                    "input": statusCBFile
+                },
+                {
+                    "query_type": "direct",
+                    "query": "Enter file name with path to log video mode update callbacks:",
+                    "input": videomodeCBFile
+                },
+                {
+                    "query_type": "direct",
+                    "query": "Enter file name with path to log allm change callbacks:",
+                    "input": allmCBFile
+                },
+                {
+                    "query_type": "direct",
+                    "query": "Enter file name with path to log audio video latency change callbacks:",
+                    "input": avlatencyCBFile
+                },
+                {
+                    "query_type": "direct",
+                    "query": "Enter file name with path to log avi content change callbacks:",
+                    "input": avicontentCBFile
+                }
+        ]
+        result = self.utMenu.select( self.testSuite, "Initialize dsHdmiIn", promptWithAnswers)
+
+    def terminate(self):
+        """
+        Terminate the HdmiIn port.
+
+        Args:
+            None.
+
+        Returns:
+            None
+        """
+        result = self.utMenu.select(self.testSuite, "Terminate dsHdmiIn")
+
+    def getports():
+        """
+        Gets the number of HdmiIn port on device.
+
+        Args:
+            None.
+        Returns:
+            bool.
+        """
+        ports = self.deviceProfile.get("Ports")
+        if not ports:
+            return false
+
+        result = self.utMenu.select( self.testSuite, "Get Input Port")
+        print(result)
```

Comment:
remove print, select prints the results to log module

Reply:
done